### DISCUSSION TITLE
set delimiter?

### DISCUSSION THREAD

**Discussion by silvacarl2** [TomWright/dasel#329] [Q&A] [answered]
is it possible to set the column delimiter used when converting from JSON to CSV?

carl

**Comment by TomWright** [comment]
Hi @silvacarl2 ,

Sorry this isn't possible at the moment but it shouldn't be too hard to implement when I get the chance.

**Reply by silvacarl2** [reply]
ok, no problem!

**Comment by TomWright** [comment, marked as answer]
Hi @silvacarl2,

This is possible from `v2.4.1`.

```
$ echo 'a;b;c
1;2;3
4;5;6' | dasel -r csv --csv-comma ';' --csv-write-comma ','
a,b,c
1,2,3
4,5,6
```

**Comment by silvacarl2** [comment]
thx will check it out!
